### ISSUE TITLE
Arch Linux GUI version stopped working

### ISSUE THREAD

**Issue by Node815** [kopia/kopia#2865]
It now shows as connecting to repository in the Status Bard Icon.

I tried running it from Command Line:

`/opt/KopiaUI/kopia-ui %U
13:07:28.196 › failed to load
13:07:28.203 › APPIMAGE env is not defined, current application is not an AppImage
libva error: /usr/lib/dri/iHD_drv_video.so init failed
13:07:28.702 › failed to load
13:07:29.205 › failed to load
13:07:29.707 › failed to load
13:07:30.209 › failed to load
13:07:30.711 › failed to load
13:07:31.213 › failed to load
13:07:31.715 › failed to load
13:07:32.218 › failed to load
13:07:32.720 › failed to load
13:07:33.222 › failed to load
13:07:33.726 › failed to load
`
Additionally, if I run it manually from the desktop, I get a KopiaUI is loading... message but white window instead.
I tested this using a fresh install of Arch Linux after discovering this as well as with an existing install.

I think it is isolated to Arch however,as I successfully  was able to run it in Fedora 37.  If there is any other log I can look at and provide I will do so as long as there is a location I can pull from which I have yet to find.

**Comment by nekorubu** [comment]
Can confirm that I'm having the same issue with a similar error on Arch Linux, even when running the AppImage itself from the command line:
`❯ ./KopiaUI-0.12.1.AppImage`
`17:00:10.947 › failed to load
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)`
`[38600:0409/170011.145661:ERROR:browser_main_loop.cc(271)] Gdk: gdk_window_thaw_toplevel_updates: assertion 'window->update_and_descendants_freeze_count > 0' failed`
Not sure if this might be due to a recent update or not.

**Comment by nekorubu** [comment]
This seems to be fixed in 0.13.0-rc2

**Comment by scaleoutsean** [comment]
I get this with 0.13 on Rocky 8.8 using kopia-ui x86_64 RPM.

The first time I installed kopia-ui 0.13 I was able to start it. After a reboot, I always get this "APPIMAGE env is not defined" error when I try from the CLI. 

![image](https://github.com/kopia/kopia/assets/16622642/44958210-025a-49e5-b094-a63d42338b15)

I removed the package with `rpm -e kopia-ui` and installed again. But it can no longer restart. 

- `kopia-ui &`: same error
- from X-Windows start menu: app starts, but no window shows up
```sh
$ ps -ef | grep kopia
sean        2937    2206  1 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui
sean        2941    2937  0 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui --type=zygote --no-zygote-sandbox
sean        2942    2937  0 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui --type=zygote
sean        2944    2942  0 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui --type=zygote
sean        2986    2937  2 07:05 ?        00:00:00 /opt/KopiaUI/resources/server/kopia server start --ui --tls-print-server-cert --tls-generate-cert-name=127.0.0.1 --random-password --random-server-control-password --tls-generate-cert --async-repo-connect --shutdown-on-stdin --address=127.0.0.1:0 --config-file /home/sean/.config/kopia/repository.config
sean        3010    2937  0 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui --type=utility --utility-sub-type=network.mojom.NetworkService --lang=en-US --service-sandbox-type=none --enable-crash-reporter=2d2f9802-7afd-4466-b10d-3500c63f441d,no_channel --user-data-dir=/home/sean/.config/kopia-ui --shared-files=v8_context_snapshot_data:100 --field-trial-handle=0,i,11044645475290708620,2839189261107904813,131072 --disable-features=SpareRendererForSitePerProcess
sean        3012    2941  0 07:05 ?        00:00:00 /opt/KopiaUI/kopia-ui --type=gpu-process --enable-crash-reporter=2d2f9802-7afd-4466-b10d-3500c63f441d,no_channel --user-data-dir=/home/sean/.config/kopia-ui --gpu-preferences=WAAAAAAAAAAgAAAIAAAAAAAAAAAAAAAAAABgAAAAAAA4AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAIAAAAAAAAAABAAAAAAAAAAgAAAAAAAAACAAAAAAAAAAIAAAAAAAAAA== --use-gl=angle --use-angle=swiftshader-webgl --shared-files --field-trial-handle=0,i,11044645475290708620,2839189261107904813,131072 --disable-features=SpareRendererForSitePerProcess
sean        3128    3070  0 07:06 pts/1    00:00:00 grep --color=auto kopia

```

It's not a big problem for me, I'll just use the Docker version, but I wanted to share that this also affects Rocky 8.8. I suspect it may be GPU driver-related (my Kopia UI runs in a VM) because I usually get these GPU-related messages after Kopia's AppImage error. Why did it work yesterday? No idea. Maybe some RPMs were auto-updated since then.

```
Sep  5 07:13:39 s198 kopia-ui.desktop[3233]: 07:13:39.990 › APPIMAGE env is not defined, current application is not an AppImage
Sep  5 07:13:40 s198 kopia-ui.desktop[3269]: [3269:0905/071340.046136:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
Sep  5 07:13:40 s198 kopia-ui.desktop[3324]: [3324:0905/071340.098419:ERROR:gpu_memory_buffer_support_x11.cc(49)] dri3 extension not supported.
```

I also tried KopiaUI-0.13.0.AppImage - the same problem in the sense that processes start but app doesn't show.

**Comment by lupusA** [comment]
Hi @scaleoutsean, 

is it solved with version 0.15? 

Cheers,

**Comment by scaleoutsean** [comment]
Not yet.

a) With Rocky 8.8 (haven't updated RPMs for a while) and Kopia-UI 0.15: nothing happens upon clicking on the Kopia icon in Start menu. Also when I run X-Windows as `root`.

b) X-Windows as a non-root user, but with `kopia-ui` started from the CLI:

![image](https://github.com/kopia/kopia/assets/16622642/1ff884d3-aac7-4884-9d71-a3edc53d3a3b)

b) Same but with `sudo strace kopia-ui --no-sandbox` I can see a blank Kopia UI. I think I couldn't before, but I am not sure.

![image](https://github.com/kopia/kopia/assets/16622642/a8b0c2cb-38ab-4f37-9d14-dc13f86fbe49)

Then I updated OS (857 new RPMs installed), disabled SELinux, and rebooted. That pushed me to Rocky 8.9 with latest updates as of Dec 11, 2023. But this resulted in no changes.
